sources/platform/actors/development/actor_definition/environment_variables.md

```diff
@@ -10,7 +10,9 @@ slug: /actors/development/environment-variables
 
 ---
 
-Aside from [custom environment variables](./source_code.md), the actor's process has several environment variables set to provide it with context:
+## System environment variables
```

Comment:
I'm not sure, but JC was talking about renaming some of those env vars to ACTOR_SOMETHING. Did it go through?